services/security-headers/security-headers.tester.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('grade of http://shields.io')
+  .get('/http/shields.io.json')
+  .expectBadge({ label: 'security headers', message: 'F', color: 'red' })
+
+t.create('grade of https://shields.io')
+  .get('/https/shields.io.json')
+  .expectBadge({ label: 'security headers', message: 'F', color: 'red' })
+
+t.create('grade when no network')
```

Comment:
This test can now go away as it's behaviour that's already tested within `BaseService`.

Reply:
Sure, completely agree!

Reply:
Done, i removed the test.